rust/main/hyperlane-base/src/contract_sync/mod.rs

```diff
@@ -120,6 +126,15 @@ where
         info!(chain = chain_name, label, "contract sync loop exit");
     }
 
+    fn update_liveness_metric(liveness_metric: &GenericGauge<AtomicI64>) {
+        liveness_metric.set(
+            UNIX_EPOCH
+                .elapsed()
+                .map(|d| d.as_millis() as i64)
```

Comment:
since this metric is only expected to be updated every secs, it's probably safer to use `d.as_secs()` instead (millis returns u256, whereas secs returns a [u64](https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_secs))

Reply:
Ok.